### DISCUSSION TITLE
How can I validate an insert/remove/update

### DISCUSSION THREAD

**Discussion by lifeiscontent** [edmundhung/conform#634] [Q&A]
Using Conform it would be really nice to be able to validate an `insert/update/remove` with the intent button functionality, e.g. validate a segment of the entire schema to be inserted/removed/updated

is this possible?

**Comment by edmundhung** [comment]
You can use `form.validate({ name: '...' })` or `form.update({ name: '...', validated: true })` to validate a field. But you have to do this one by one. What is your usecase?

**Reply by lifeiscontent** [reply]
@edmundhung basically, I want do a validation for a segment of a schema, e.g. I have a form drawer where I show the individual item to be added to the user, and if its valid during there button press, do the insert
